pkg/pro/platform.go

```diff
@@ -7,6 +7,6 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
-var ConnectToPlatform = func(context.Context, *config.VirtualClusterConfig, manager.Manager) error {
-	return nil
+var ConnectToPlatform = func(context.Context, *config.VirtualClusterConfig) (func(mgr manager.Manager) error, error) {
+	return nil, nil
```

Comment:
Return a function that does nothing here to avoid lint and avoid the nil check above